lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -764,18 +764,29 @@ static void addSendableFixIt(
   }
 }
 
-/// Add Fix-It text for the given generic param declaration type to adopt
-/// Sendable.
-static void addSendableFixIt(const GenericTypeParamDecl *genericArgument,
-                             InFlightDiagnostic &diag) {
+static void addProtocolFixIt(const GenericTypeParamDecl *genericArgument,
+                             InFlightDiagnostic &diag,
+                             std::string protocolName) {
   if (genericArgument->getInherited().empty()) {
     auto fixItLoc = genericArgument->getLoc();
-    diag.fixItInsertAfter(fixItLoc, ": Sendable");
+    diag.fixItInsertAfter(fixItLoc, ": " + protocolName);
   } else {
     auto fixItLoc = genericArgument->getInherited().getEndLoc();
-    diag.fixItInsertAfter(fixItLoc, " & Sendable");
+    diag.fixItInsertAfter(fixItLoc, " & " + protocolName);
   }
 }
+/// Add Fix-It text for the given generic param declaration type to adopt
+/// Sendable.
+static void addSendableFixIt(const GenericTypeParamDecl *genericArgument,
+                             InFlightDiagnostic &diag) {
+  addProtocolFixIt(genericArgument, diag, "Sendable");
```

Comment:
minor nit, is it worth doing `getProtocolName(KnownProtocolKind::Sendable)` ?

Reply:
Oh, good idea!